packages/javascript/bh-shared-ui/src/utils/permissions.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+export enum Permission {
+    APP_READ_APPLICATION_CONFIGURATION,
+    APP_WRITE_APPLICATION_CONFIGURATION,
+    APS_GENERATE_REPORT,
+    APS_MANAGE_APS,
+    AUTH_ACCEPT_EULA,
+    AUTH_CREATE_TOKEN,
+    AUTH_MANAGE_APPLICATION_CONFIGURATIONS,
+    AUTH_MANAGE_PROVIDERS,
+    AUTH_MANAGE_SELF,
+    AUTH_MANAGE_USERS,
+    CLIENTS_MANAGE,
+    CLIENTS_READ,
+    CLIENTS_TASKING,
+    COLLECTION_MANAGE_JOBS,
+    GRAPH_DB_READ,
+    GRAPH_DB_WRITE,
+    SAVED_QUERIES_READ,
+    SAVED_QUERIES_WRITE,
+    FAKE_PERMISSION,
```

Comment:
Maybe we can remove this one? Doesn't seem to be used in a test or anything

Reply:
done!